### PR TITLE
add realtime SSE connection

### PR DESCRIPTION
**Pull request by ajwootto**

### CODE REVIEW

File: DevCycle.podspec [hunk 1]
```diff
@@ -18,4 +18,6 @@ Pod::Spec.new do |spec|
   spec.source_files  = "DevCycle/**/*.{h,m,swift}"
   spec.swift_version = '5.0'
 
+  spec.dependency 'LDSwiftEventSource', '~> 3.0'
```
Comment (jsalaber) [inline DevCycle.podspec:21]: also gotta make sure people that the library can be bundled with the framework even if DevCycle is installed with Swift Package Manager and Carthage

**Review by jsalaber — CHANGES_REQUESTED** [review]

**Comment by ajwootto** [comment]
Current dependencies on/for this PR:
* main
  * **PR #83** <a href="https://app.graphite.dev/github/pr/DevCycleHQ/ios-client-sdk/83" target="_blank">Graphite</a>  👈
    * **PR #85** <a href="https://app.graphite.dev/github/pr/DevCycleHQ/ios-client-sdk/85" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/DevCycleHQ/ios-client-sdk/83?utm_source=stack-comment).

File: Podfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
+
+target 'DevCycle' do
+  # Comment the next line if you don't want to use dynamic frameworks
+  use_frameworks!
+
+  # Pods for DevCycle
+  pod 'LDSwiftEventSource', '~> 3.0'
```
Comment (jsalaber) [inline Podfile:9]: if you're using carthage in the example app, do we still need the `Podfile` and `Podfile.lock` here?

**Review by jsalaber — CHANGES_REQUESTED** [review]

File: Podfile (hunk 2 above)
Comment (ajwootto) [reply Podfile:9]: I was using cocoapods in the example app

Comment (jsalaber) [reply Podfile:9]: the example app has it's own Podfile so we don't need this one

Comment (ajwootto) [reply Podfile:9]: oh this is the root one, thats weird I thought I removed this..

**Review by jsalaber — APPROVED** [review]

**Pull request merged**